001-core/privacy.php

```diff
@@ -457,3 +457,22 @@ function delete_old_export_files() {
 		}
 	}
 }
+
+/**
+ * Disable crawling for go-vip.co and go-vip.net domains.
+ * 
+ * @param string $output The robots.txt content.
+ * @return string The modified robots.txt content.
+ */
+function vip_convenience_domain_robots_txt( $output ) {
+	$host = strtolower( $_SERVER['HTTP_HOST'] ?? '' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Is `$_SERVER['HTTP_HOST']` set for curl requests?

I see the nullcoalescing, but want to avoid customers making requests with curl and a browser on their convenience domain sites and getting different results.

Reply:
I think so, unless the host parameter is set otherwise?